<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QDialog" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PoolMate Viewer 0.02</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="technique">
       <attribute name="title">
        <string>Technique</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GraphWidget" name="graphWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="volume">
       <attribute name="title">
        <string>Volume</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="GraphWidget" name="volumeWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="bottomLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="workoutGrid">
           <property name="minimumSize">
            <size>
             <width>502</width>
             <height>0</height>
            </size>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="rowCount">
            <number>1</number>
           </property>
           <property name="columnCount">
            <number>8</number>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>19</number>
           </attribute>
           <row/>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pool</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Duration</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cal</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Lengths</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Total m</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Rest</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="5">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="6">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="7">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonLayout2">
           <item>
            <widget class="QPushButton" name="editButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Graph:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="viewCombo">
             <item>
              <property name="text">
               <string>Workouts</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Weekly</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Monthly</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Year by week</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Year by month</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="setGrid">
           <property name="minimumSize">
            <size>
             <width>501</width>
             <height>0</height>
            </size>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>19</number>
           </attribute>
           <column>
            <property name="text">
             <string>Set</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Duration</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Strokes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Dist</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Speed</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Effic</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Rate</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Stroke</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="calendarLayout">
         <item>
          <widget class="CalendarWidget" name="calendarWidget">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="firstDayOfWeek">
            <enum>Qt::Monday</enum>
           </property>
           <property name="gridVisible">
            <bool>true</bool>
           </property>
           <property name="horizontalHeaderFormat">
            <enum>QCalendarWidget::ShortDayNames</enum>
           </property>
           <property name="verticalHeaderFormat">
            <enum>QCalendarWidget::NoVerticalHeader</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Today</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="downloadButton">
             <property name="text">
              <string>Download...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="configButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Config...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="text">
              <string>Close</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header location="global">calendar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>workoutGrid</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Summary</receiver>
   <slot>workoutSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>showToday()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewCombo</sender>
   <signal>activated(int)</signal>
   <receiver>Summary</receiver>
   <slot>scaleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>Summary</receiver>
   <slot>selectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>syncButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>791</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>editButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>deleteButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>configButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>832</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedDate(QDate)</slot>
  <slot>selectedWorkout(int,int)</slot>
  <slot>startDate(QDate)</slot>
  <slot>endDate(QDate)</slot>
  <slot>syncButton()</slot>
  <slot>configButton()</slot>
  <slot>deleteButton()</slot>
  <slot>editButton()</slot>
  <slot>display()</slot>
  <slot>workoutSelected()</slot>
  <slot>scaleChanged(int)</slot>
 </slots>
</ui>
